_layouts/frontpage.html

```diff
@@ -15,13 +15,13 @@ <h1>{{site.country.name}} {{ t.frontpage.heading }}</h1>
         {% cycle 'add row' : '<div class="row no-gutters">', '', '', '', '', '' %}
             <div class="col-xs-4 col-md-2">
                 <a href="./{{ default_goal.short | slugify }}/">
-                  <img src="{{ site.baseurl }}/assets/img/goals/{{ current_language }}/{{ goal_number }}.png" id="goal-{{ goal_number }}" alt="{{ translated_goal.short }} - {{ t.general.goal }} {{ goal_number }}" />
+                  <img src="https://open-sdg.github.io/sdg-translations/assets/img/goals/{{ current_language }}/{{ goal_number }}.png" id="goal-{{ goal_number }}" alt="{{ translated_goal.short }} - {{ t.general.goal }} {{ goal_number }}" />
```

Comment:
Would it be better to have this as a parameter? I even wonder if a whole translations section of the `_config.yml` would be useful.